README.md

```diff
@@ -1 +1,22 @@
 # opentelemetry-log-collection
```

Comment:
Can we add a notice that the repository is unstable and should not be used outside OpenTelemetry? I want to make sure we are free to make any changes we want and do not break someone who is unaware of what the intent is.

Reply:
Added a note.